x-pack/plugins/security_solution/public/detections/configurations/security_solution_detections/render_cell_value.tsx

```diff
@@ -173,10 +174,11 @@ export const getRenderCellValueHook = ({
             scopeId={tableId}
             truncate={truncate}
             asPlainText={false}
+            context={context as RenderCellValueContext}
```

Comment:
We've had bugs before when casting props. Is there any way around the cast here? Are there any values that could be missing in `context` that are needed in `RenderCellValue`?

Reply:
The idea was to make `triggers_action_ui` agnostic to context format. That's why we put first `any` but then switched to `unknown`. This makes us do this casting on our side, since we create the context and know exact type.